ops/terraform/services/pipeline/modules/bfd_pipeline_manager/README.md

```diff
@@ -0,0 +1,73 @@
+# `bfd_pipeline_manager` Sub-module
+
+This submodule defines a Lambda and its corresponding infrastructure. This Lambda,
+`bfd-${env}-pipeline-manager`, "manages" the BFD Pipeline Terraservice; it is invoked when new files
+are created in the `env`'s corresponding S3 pipeline/ETL bucket at specific paths:
+
+- `Synthetic/Incoming/`
+- `Synthetic/Done/`
+- `Incoming/`
+- `Done/`
+
+Objects uploaded to or moved to these paths in the ETL bucket will invoke this Lambda, which will
+then determine if the BFD CCW Pipeline should be started or not. This depends on a few factors:
+
+- If the file was uploaded to either of the `Incoming` folders, and there are no _ongoing_ or
```

Comment:
Mitch, are there any safeguards to preclude the Pipeline from firing prematurely; for example, there is an 'implied' prerequisite that pipeline should not start unless/until the manifest(s) is/are available. So CCW provides a manifest per bene/claims file while synthea provides a single manifest file for all data files. In the case of CCW, their 'delivery' contract requires them to provide data files prior to delivering a manifest file for that data.

Reply:
Since this solution doesn't modify the Pipeline itself, we'll be relying on the Pipeline's built-in safeguards to ensure that the Pipeline does not start loading data prematurely. The Lambda _does_ filter based on _what_ file invoked it; for example, if the file uploaded to any of the watched folders in S3 is not a valid RIF file, the Lambda immediately exits.

Basically, you can boil the Lambda down to a mechanism for starting and stopping the Pipeline's _instance_ depending on when we need it available.